tests/plugins/widget/manual/widgetwithspaces.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.8.0, bug, 605
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, wysiwygarea, toolbar, sourcearea
+
+**Scenario:**
+1. Click `Source` button.
+2. Paste `lorem<span> ipsum&nbsp;<strong>dolor sit </strong></span>amet` into source area.
```

Comment:
Can we wrap `lorem<span> ipsum&nbsp;<strong>dolor sit </strong></span>amet` inside `textarea` or `input`? On Edge it is impossible to select without content around which makes executing this test very annoying ;)

Reply:
Of course, we can :)